service/SimpleParsed.java

```diff
@@ -0,0 +1,39 @@
+package service;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+
+import model.ParsedInterface;
+
+
+public class SimpleParsed implements ParsedInterface {
```

Comment:
@undsett Why not FileParsed? Since source of a content is file for this implementation, right?

Reply:
agree